### DISCUSSION TITLE
Success

### DISCUSSION THREAD

**Discussion by Jens5395** [emporia-vue-local/esphome#5] [General]
I am happy to report that changing the configuration "time" to

```
time:
  - platform: homeassistant
    id: homeassistant_time
```

resulted in the totals resetting at midnight as they were supposed to.
Thank you mgulick !

**Comment by weltmeyer** [comment]
hmm...
for me, that results in:
`*** [/data/emporiavue2_1/.pioenvs/emporiavue2_1/src/esphome/components/sntp/sntp_component.o] Source `src/esphome/components/sntp/sntp_component.cpp' not found, needed by target `/data/emporiavue2_1/.pioenvs/emporiavue2_1/src/esphome/components/sntp/sntp_component.o'.`

**Reply by flaviut** [reply]
Take a look at https://github.com/emporia-vue-local/esphome/discussions/20
